tests/test_enterprise/api/test_views.py

```diff
@@ -2416,6 +2416,8 @@ def test_reporting_config_post_permissions(self, has_feature_role, expected_stat
         if has_feature_role:
             response_content = self.load_json(response.content)
             assert response_content['enterprise_customer']['uuid'] == str(enterprise_customer.uuid)
+            assert response_content['encrypted_password'] is not None
```

Comment:
I'm understanding this as `encrypted_password` sent to the API is unencrypted "password" and the `encrypted_password` in the response should be encrypted `q1w2e3r4`. If true, I think it's at least worth verifying that the response value is not equal to the submitted value.

You could go farther by encrypted the submitted value manually and comparing, but that seems like testing the framework, which isn't necessary.

Reply:
I also just realized that this is really a permissions test. I'd understand if this isn't the place to put that test.